message MSG_CLIENT_DM_USER_DESTORY
{
    optional string token = 1;
    optional string userName = 2;
}

message MSG_DM_CLIENT_USER_DESTORY_ACK
{
    optional int32 statusCode = 1;
}